scripts/conv_frames.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Convert the watson frames file from the old to new format and vice versa."""
+
+from __future__ import print_function
+
+import json
+import os
+import tempfile
+
+import click
+
+
+app_dir = click.get_app_dir('watson')
```

Comment:
I added support for this in PR #115, from which this port is derived. But please note that I wrote this script only for the convenience of developers, switching between old and new versions of Watson. The `frames` module, as implemented in PR #115 is backwards-compatible, it  detects a frame file in the old format automatically and converts it while loading it.